walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/script.js

```diff
@@ -70,4 +70,4 @@ async function getRandomQuoteUsingAsyncAwait() {
   const response = await fetch('/random-quote');
   const quote = await response.text();
   document.getElementById('quote-container').innerText = quote;
-}
+}
```

Comment:
This file and ServerStatsServlet.java both lost their end-of-file newlines in this PR. It's not a really meaningful change, and probably happened inadvertently when they got opened in some editor and possibly re-saved. That happens all the time.

However, we should avoid including changes in a commit that don't contribute to the purpose of the commit. Concretely, in cases like this, if I noticed an unexpected file in the list of files that git shows me when writing a commit comment, I would cancel the commit process, revert the unnecessary changes, and then start the commit process again. In fact, my usual workflow is usually to review a full diff of my workspace compared to the last commit before I initiate a commit, so that I can ensure I'm only committing what I intend.

It's not a huge difference in this case, so I'll leave it up to you whether you want to add back the newline to these files in a future commit in this branch.